src/lwc/psElevateTokenHandler/psElevateTokenHandler.js

```diff
@@ -144,23 +143,20 @@ class psElevateTokenHandler {
     }
 
     /***
-    * @description Method sends a message to the visualforce page iframe requesting a token.
-    * This request response is found and handled in the registerPostMessageListener().
-    * @param iframe The payment services iframe displayed within the credit card widget LWC
-    * @param cardholderName The cardholder name
-    * @param handleError An error handler function
-    * @param resolveToken Function (if any) called when a token is generated
-    * @return Promise A token promise
-    */
-    requestToken(iframe, cardholderName, handleError, resolveToken) {
-        if (isNull(iframe)) {
+     * @description Method sends a message to the visualforce page iframe requesting a token.
+     * This request response is found and handled in the registerPostMessageListener().
+     * @param {Object} params An object that holds parameters needed to tokenize credit card and ACH transactions
+     * @return Promise A token promise
+     */
+    requestToken(params) {
+        if (isNull(params.iframe)) {
```

Comment:
It is an improvement to pass an object here but the change makes it a bit more difficult to figure out what properties of the passed in object the function is using/expecting.  

What would you think about using a destructured object as the param to increase self documentation, provide a default value (empty object), and allow for removal of the qualifying "params." when referencing the args in the function body? 

```javascript
requestToken({  
                     iframe,  
                     cardholderName,  
                     handleError,
                     resolveToken,
                     eventAction,
                     tokenizeParameters
                 } = {}) {

        if (isNull(iframe)) {
            return;
        } ...
```

Reply:
@kenlewis-sfdo I am not fully sold on us using a destructured object as the function argument, this function is called in two places and I believe we're passing a destructured version of the params object from both places. So we should be able to tell what is being passed down. @psadie-sfdo What are your thoughts on this?

Reply:
I like it! I was about to request that we call out all the properties that `params` should have in the jsdoc.

I hear your point in that we only call this from two places, but if I was a first-time reader of this method I'd much prefer immediately being able to tell what needs to passed in instead of investigating call sites for `requestToken` to see what others have done before.

The self-docing is ace 🙏

Reply:
Thanks! @kenlewis-sfdo and @psadie-sfdo. I'll make that change now.